amlip_docs/rst/user_manual/tools/agent.rst

```diff
@@ -0,0 +1,188 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_tools_agent:
+
+##########
+Agent Tool
+##########
+
+This tool executes an :ref:`user_manual_nodes_agent` which is the main transport component that manages the dynamic discovery of *AML-IP Algebraic Nodes* deployed within a :term:`LAN` and the discovery of other AML-IP clusters over :term:`WAN`.
+
+Building the tool
+=================
+
+If the tool package is not compiled, please refer to :ref:`developer_manual_installation_sources_linux_colcon` or run the command below.
+
+.. code-block:: bash
+
+    colcon build --packages-up-to-regex amlip
```

Comment:
amlip_agent??

Reply:
It is a regexp, so it does not really matter.